lib/Email/MIME.pm

```diff
@@ -119,7 +119,15 @@ my $NO_ENCODE_RE = qr/
 /ix;
 
 sub new {
-  my $self = shift->SUPER::new(@_);
+  my ($class, $text, $arg, @rest) = @_;
+  $arg ||= {};
+
+  my $encode_check = exists $arg->{encode_check}
+                   ? delete $arg->{encode_check}
+                   : 1; # Encode::FB_CROAK
```

Comment:
I do not know about any reason why to not use `Encode::FB_CROAK` today. Maybe at that time `Encode::FB_CROAK` was not defined or author did not know about `Encode::FB_CROAK`. But Email::MIME already depends on Encode 1.98 which `Encode::FB_CROAK` [supports](https://metacpan.org/pod/release/DANKOGAI/Encode-1.98/Encode.pm#Handling-Malformed-Data).